lib/graphql-docs/layouts/graphql_enums.html

```diff
@@ -1,4 +1,4 @@
-<h1><%= type[:name] %></h1>
+<h1 class="anchored"><%= type[:name] %></h1>
```

Comment:
Since `h1`s are never anchored, I'm not sure I am a fan of adding the class here. (I'm not sure what someone's CSS logic around `anchored` would do that could negatively effect this.) Would you mind taking the `h1` classes out? 🙏

Reply:
Ah, I might have been confusing this with developer.github.com, which _does_ anchor `h1` tags: https://developer.github.com/v4/enum/. In either case, I'll remove this.